README.md

```diff
@@ -146,3 +146,16 @@ Tests can be run using [npm][npm]:
 [witten-markets]:http://www.stadtmarketing-witten.de/einkaufen/wochenmaerkte.html
 [wuppertal-wikipedia]:https://en.wikipedia.org/wiki/Wuppertal
 [wuppertal-markets]:https://www.wuppertal.de/tourismus-freizeit/einkaufen/102370100000204430.php
+
+
+## Deployment
```

Comment:
Please consolidate the wording - I recommend using `Travis CI` not `travis` or `Travis`.